app/client/components/radioButton.tsx

```diff
@@ -0,0 +1,51 @@
+import { css } from "@emotion/core";
+import React, { ChangeEvent } from "react";
+import palette from "../colours";
+
+export interface RadioButtonProps {
+  key: any;
+  value: any;
+  label: string;
+  checked: boolean;
+  onChange: (event: ChangeEvent<HTMLInputElement>) => void;
+  groupName: string;
+}
+
+export const RadioButton = (props: RadioButtonProps) => (
+  <label
+    css={css({
+      display: "block",
+      cursor: "pointer",
+      marginBottom: "7px"
+    })}
+  >
+    <input
+      type="radio"
+      value={props.value}
+      name={props.groupName}
+      onChange={props.onChange}
+      checked={props.checked}
+      css={{ display: "none" }}
+    />
+    <div css={{ position: "relative" }}>
+      <div
```

Comment:
Is this component rendering an alternative radio button (o) to the <input> above??

Reply:
yes

Reply:
Let's just double check it works OK with some accessibility tools / Lynx etc.

Reply:
Have now made the radio buttons accessible.